defaults/main.yaml

```diff
@@ -12,6 +12,15 @@ clamav_epel_yumrepo_gpgkey: "{{ yumrepo_epel_gpgkey | default('https://dl.fedora
 # Additional EPEL YUM repo params
 clamav_epel_yumrepo_params: "{{ yumrepo_epel_params | default({}) }}"
 
+# In the event of network congestion
+clamav_network_max_retries: 0
+
+# ClamAV operating as client-only
+clamav_client_only: false
+
+# Queried clamav version in tasks/main.yml
+clamav_version: 0.999.0
```

Comment:
This is not needed if you are querying the version in in the task.

Reply:
Removed